src/collector.ts

```diff
@@ -221,8 +222,12 @@ export const collector = async ({
 
   output.uri_dest = page.url();
   duplicatedLinks = await getLinks(page);
+
+  const sitemapperSites = await retrieveSites(page.url())
+  let joinedDuplicatedLinks = [...duplicatedLinks, ...sitemapperSites]
```

Comment:
What's the purpose of creating a new variable here? Couldn't you just reassign the `duplicatedLinks` again?

```suggestion
  duplicatedLinks = await getLinks(page);

  const sitemapperSites = await retrieveSites(page.url())
  duplicatedLinks = [...duplicatedLinks, ...sitemapperSites]
```

Or maybe even

```suggestion
  duplicatedLinks = [...(await getLinks(page)), ...(await retrieveSites(page.url()))]
```

Or possibly 

```suggestion
  duplicatedLinks = (await getLinks(page)).concat((await retrieveSites(page.url())))
```

Reply:
I am trying to leave the old code in place and make it clear what I am adding in.  There is always ways to do multiple things.  The last way you suggested definitely reduces lines of code.

Reply:
I hate re-assigning so generally argue against it.

Also, in this particular case, the `let` can be a `const` -- while the array is being mutated in line 275, there doesn't seem to be any reason to do a concat/reassign vs `push`ing the new links in.